admin_manual/configuration/server/security/ransomware-protection.rst

```diff
@@ -16,32 +16,64 @@ About Ransomware Protection
+Like other forms of cyberattack, ransomware has a range of diverse characteristics.
+On the one hand it makes them hard to detect, and on the other it makes them even harder to prevent.
+Recent ransomware attacks either encrypt a user's files and add a specific file extension to them (e.g., ".crypt"), or they replace the original files with an encrypted copy and add a particular file extension.
+
+File Extension Blacklist
+^^^^^^^^^^^^^^^^^^^^^^^^
+
+The first line of defense against such threats is a blacklist that blocks write access to file extensions known to originate from ransomware.
+
+.. note:: 
+   Write access (e.g., moving and deleting files) is still possible for users, when they log in with their web browser.
 
-The first line of defense against such threats is a blacklist that blocks uploading files with file extensions known to originate from ransomeware.
 Ransomware Protection ships with `a static extension list`_ of around 1,500 file extensions.
-As new extensions are regularly created this list needs maintenance.
+As new extensions are regularly created, this list also needs to be regularly reviewed and updated.
 Future releases of Ransomware Protection will include an updated list and the ability to update the list via syncing with `FSRM's API`_ by using :doc:`occ <../occ_command>`.
 
-.. _ransomeware_protection_label:
+.. important:: 
+   Please check the provided ransomware blacklist!
+   It is **strongly recommended** to check the provided ransomware blacklist to ensure that it fits your needs. 
+   In some cases, the patterns might be too generic and result in false positives.
+
+Account Locking
+^^^^^^^^^^^^^^^
+
+The second line of defense is account locking. 
+If a client uploads a file which matches a pattern in the ransomware blacklist, the account will be locked (set as read-only) for client access (*create*, *change*, *move*, and *delete* operations) to prevent further malicious changes. 
+
+Following this, clients receive an error (403 Access Forbidden) which notifies the user that the account is locked by Ransomware Protection.
+
+When an account is locked, administrators can unlock the account using the ``occ ransomguard:unlock`` command.
+Administrators can also manually lock user accounts, using the ``occ ransomguard:lock`` command.
+
+.. note:: 
+   When an account is locked, it will still be fully usable from the ownCloud web UI.
+   However, ownCloud clients (as well as other WebDAV clients) will see the account as set to read-only mode.
+
+Users will see a yellow notification banner in the ownCloud web UI directing them to "Personal Settings -> Security" ("*Locked by Ransomware Protection app. Unlock your account in the security settings panel.*"), where additional information is displayed and users can unlock their account when ransomware issues are resolved locally.
```

Comment:
Final decision for the notification banner wording: "Ransomware detected: Your account is locked (read-only) for client access to protect your data. Click here to unlock."

Reply:
I realise it's the final wording, but how about yet one more suggestion: 

> Ransomware has been detected. Your account is locked (set as read-only for client access) to protect your data. Click here to unlock your account.

Reply:
Thanks for your thoughts. As we need to keep this as short as possible for that small notification banner I would leave it as it is.

Reply:
Hmm, ok, let's take this one: "Ransomware detected: Your account is locked (read-only for client access) to protect your data. Click here to unlock."